drivers/flash/flash_stm32g4x.c

```diff
@@ -176,6 +207,29 @@ int flash_stm32_write_range(const struct device *dev, unsigned int offset,
 
 	return rc;
 }
+#ifdef FLASH_OPTR_DBANK
+void flash_stm32_page_layout(const struct device *dev,
```

Comment:
second implementation of changes should flash_stm32_page_layout should be noticed in commit message.